doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,230 @@
+        "tag"
+      ]
+    },
+    "kubernetes": {
+      "type": "object",
+      "description": "The kubernetes-level metadata",
+      "properties": {
+        "minimumKubernetesVersion": {
+          "type": "string",
+          "description": "Minimum required version of kubernetes needed to run this connector"
+        }
+      }
+    },
+    "architecture": {
+      "type": "string",
+      "description": "The architecture that this connector should run on"
+    },
+    "sourceCode": {
+      "type": "object",
+      "description": "Details about the source code that compiled this connector",
+      "properties": {
+        "language": {
+          "type": "string",
+          "description": "The programming language used to write this connector"
+        },
+        "languageVersion": {
+          "type": "string",
+          "description": "The version of the programming language used to write this connector"
+        },
+        "sdk": {
+          "type": "object",
+          "description": "Details about the versions of the AIO SDKs used to write this connector (if they were used)",
+          "properties": {
+            "mqttPackageVersion": {
+              "type": "string",
+              "description": "The version of the azure iot operations MQTT package"
+            },
+            "servicesPackageVersion": {
+              "type": "string",
+              "description": "The version of the azure iot operations services package"
+            },
+            "connectorPackageVersion": {
+              "type": "string",
+              "description": "The version of the azure iot operations connector package"
+            }
+          }
+        }
+      },
+      "required": [
+        "language",
+        "languageVersion"
+      ]
+    },
+    "resourceRequirements": {
+      "type": "object",
+      "description": "The minimal resource requirements to run this connector",
+      "properties": {
+        "cpu": {
+          "type": "string"
+        },
+        "memory": {
+          "type": "string"
+        },
+        "storage": {
+          "type": "string"
+        }
+      }
+    },
+    "aioConfig": {
+      "type": "object",
+      "description": "The AIO requirements to run this connector",
+      "properties": {
+        "minVersion": {
+          "type": "string",
+          "description": "The minimal (inclusive) version that can run this connector."
+        },
+        "maxVersion": {
+          "type": "string",
+          "description": "The maximal (inclusive) version that can run this connector."
+        }
+      }
+    },
+    "protocols": {
+      "type": "array",
+      "description": "The array of protocols that this connector operates over.",
+      "items": [
+        {
+          "type": "object",
+          "properties": {
+            "name": {
+              "type": "string"
+            },
+            "version": {
+              "type": "string"
+            },
+            "configSchemas": {
```

Comment:
```
properties:
  configurationSchemaRefs:
    nullable: true
    properties:
      additionalConfigSchemaRef:
        nullable: true
        type: string
      defaultDatasetConfigSchemaRef:
        nullable: true
        type: string
      defaultEventsConfigSchemaRef:
        nullable: true
        type: string
      defaultProcessControlConfigSchemaRef:
        nullable: true
        type: string
      defaultStreamsConfigSchemaRef:
        nullable: true
        type: string
    type: object
```

Reply:
I thought so, based on the naming of this field: https://github.com/Azure/iot-operations-sdks/blob/47a20a5662b5f1579c7c1d1cdddb9d218211dbd2/eng/dtdl/device-name-based-operations.json#L1024

If this isn't another "stringified json" field though, I'd be happy to remove it from this schema

Reply:
Let me get back to you on this.